src/algebra/ring/aut.lean

```diff
@@ -60,4 +64,38 @@ by refine_struct { to_fun := ring_equiv.to_mul_equiv }; intros; refl
 def to_perm : ring_aut R →* equiv.perm R :=
 by refine_struct { to_fun := ring_equiv.to_equiv }; intros; refl
 
+end mul_add
+
+section semiring
+
+variables (R G : Type*) [group G] [semiring R]
+
+/-- The tautological action by the group of automorphism of a ring `R` on `R`.-/
+instance apply_mul_semiring_action :
+mul_semiring_action (ring_aut R) R :=
```

Comment:
This doesn't need line-wrapping I don't think
```suggestion
instance apply_mul_semiring_action : mul_semiring_action (ring_aut R) R :=
```
(If it's over 100 chars, then it should be:)
```suggestion
instance apply_mul_semiring_action :
  mul_semiring_action (ring_aut R) R :=
```